UPGRADE-4.1.md

```diff
@@ -16,3 +16,8 @@ Translation
 
  * The `FileDumper::setBackup()` method is deprecated and will be removed in 5.0.
  * The `TranslationWriter::disableBackup()` method is deprecated and will be removed in 5.0.
+
+FrameworkBundle
```

Comment:
Sorry for nitpicking @Simperfit, but could you please move this up sort entries alphabetically (makes searching for something much easier in the future with many more entries)?

Reply:
done